spec/models/solidus_volume_pricing/pricer_spec.rb

```diff
@@ -0,0 +1,280 @@
+require 'spec_helper'
+
+RSpec.shared_examples 'having the variant price' do
```

Comment:
This is fine. 

Typically though, we usually find specs one place where DRY doesn't always apply.

The benefit of using these for only one small example such as this is outweighed by the fact we have to move around the file to ascertain what's actually being run. If we had a large number of shared examples this becomes more worthwhile, but for a reader, it's probably more helpful to have it defined inline.